Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml

```diff
@@ -0,0 +1,1268 @@
+    type: title
+    task:
+      id: f2dbaff5-7c92-47ad-80cc-991bfd80ff98
+      version: -1
+      name: Endpoint Reputation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "36"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -790,
+          "y": 580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 50fed99c-1eb9-4a6f-85d0-f9d5ee74bc5a
+    type: condition
+    task:
+      id: 50fed99c-1eb9-4a6f-85d0-f9d5ee74bc5a
+      version: -1
+      name: Should use !endpoint command?
+      description: Check if should run endpoint reputation command
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "37"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.UseReputationCommand
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -790,
+          "y": 720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: 79880978-7c28-4aaf-83b3-7f8ea42da67f
+    type: regular
+    task:
+      id: 79880978-7c28-4aaf-83b3-7f8ea42da67f
+      version: -1
+      name: Check Reputation
+      description: Returns information about an endpoint.
+      script: '|||endpoint'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      hostname:
+        complex:
+          root: inputs.Hostname
+      id:
+        complex:
+          root: inputs.EndpointID
+      ip:
+        complex:
+          root: inputs.IPAddress
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1000,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "38":
+    id: "38"
+    taskid: 84483658-9591-43b1-8cc6-89df338d6bd8
+    type: regular
+    task:
+      id: 84483658-9591-43b1-8cc6-89df338d6bd8
+      version: -1
+      name: Crowdstrike Search device
+      description: Searches for a device that matches the query.
+      script: CrowdstrikeFalcon|||cs-falcon-search-device
```

Comment:
Use cs-falcon-search-device instead of cs-falcon-search-device (CrowdstrikeFalcon)
Don't use the specific integration command.

Reply:
Done